.size-limit.js

```diff
@@ -27,13 +27,13 @@ module.exports = [
     name: 'The size of all the modules of material-ui.',
     webpack: true,
     path: 'packages/material-ui/build/index.js',
-    limit: '95.4 KB',
+    limit: '99.5 KB',
```

Comment:
I have one concern here 4kB for supporting a new variant? Something is wrong. Especially if people don't use this variant.